docs/how.md

```
@@ -271,6 +271,37 @@ print(Squash.vlq().des(cursor))
+local cursor = Squash.cursor()
+Squash.range(100, 150).ser(cursor, 130)
+Squash.print(cursor)
+-- Pos: 1 / 8
+-- Buf: { 30 0 0 0 0 0 0 0 }
+--           ^
+local buf = Squash.tobuffer(cursor)
+
+local cursor = Squash.frombuffer(buf)
+print(Squash.range(100, 150).des(cursor))
+-- 130
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)